pkg/telemetry/stats_test.go

```diff
@@ -43,7 +44,10 @@ func Test_ParticipantAndRoomDataAreSentWithAnalytics(t *testing.T) {
 	packet := 33
 	stat := &livekit.AnalyticsStat{Streams: []*livekit.AnalyticsStream{{PrimaryBytes: uint64(packet)}}}
 	fixture.sut.TrackStats(livekit.StreamType_DOWNSTREAM, partSID, "", stat)
-	fixture.sut.SendAnalytics()
+
+	// flush
+	time.Sleep(time.Millisecond * 500)
```

Comment:
nit: can there be a helper function for this? instead of copy & pasted sleep + flush call